article-analyzer/src/main/java/com/reader/article_analyzer/Model/User.java

```diff
@@ -0,0 +1,66 @@
+package com.reader.article_analyzer.Model;
+
+public class User {
+    private Long id;
+    private String username;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    public String getSlackId() {
+        return slackId;
+    }
+
+    public void setSlackId(String slackId) {
+        this.slackId = slackId;
+    }
+
+    private String email;
+    private String slackId;
+    private NotificationType notificationType;
```

Comment:
keep all fields together and all methods together, do not mix them

Reply:
Got it. It was an accident